src/main/scala/io/viash/engines/DockerEngine.scala

```diff
@@ -206,26 +215,26 @@ final case class DockerEngine(
     val entrypointStr = Docker.listifyOneOrMore(entrypoint).map(s => s"\nENTRYPOINT $s").getOrElse("")
     val cmdStr = Docker.listifyOneOrMore(cmd).map(s => s"\nCMD $s").getOrElse("")
 
-    s"""FROM $fromImageInfo$entrypointStr$cmdStr 
+    s"""FROM $fromImageInfo$entrypointStr$cmdStr
         |${runCommands.mkString("\n")}
-        |""".stripMargin           
+        |""".stripMargin
   }
 
-  def getTargetRegistryWithFallback(config: Config): Option[String] = {
-    target_registry.orElse(config.links.docker_registry)
-  }
+  def getTargetIdentifier(config: Config): DockerImageInfo = {
+
+    val targetRegistryWithFallback = target_registry.orElse(config.links.docker_registry)
```

Comment:
If I look at this PR, would it then also not make sense to remove the component config > links > docker_registry and change this line to:

```suggestion
    val targetRegistryWithFallback = target_registry.orElse(config.flatMap(_.links.docker_registry)).filter(_.nonEmpty)
```

?

WDYT?

Reply:
Surely it should be
```
val targetRegistryWithFallback = target_registry.orElse(config.links.docker_registry).filter(_.nonEmpty)
```
then. Config isn't an Option[...] here.
But sure, we can add this. The overriding of package config values is slightly different, but in the end, not that totally different.

Reply:
:+1: